### PR TITLE
Natasha/search bar

### PR DESCRIPTION
**Pull request by natashaarmbrust**
create search bar for liked, followers, and following

### CODE REVIEW

**Comment by lucasderraugh** [comment]
Ok, I rebased to resolve conflicts with master. I know we still have the color behind the liked bar issue, but I think it's worth bringing the branch in. One final change I'd like you to make before merging into master is that the UsersVC should jump out of the search mode before going to one of the selected users. I don't want to have to hit the cancel button if I go back to this view because now the back button is covered by the search.

File: IceFishing/Controllers/LikedTableViewController.swift [hunk 1]
```diff
@@ -8,20 +8,45 @@
...
+		
+		searchController = UISearchController(searchResultsController: nil)
+		searchController.dimsBackgroundDuringPresentation = false
+		searchController.delegate = self
+		searchController.searchResultsUpdater = self
+		
+		//Formating for search Bar
+		searchController.searchBar.sizeToFit()
+		searchController.searchBar.delegate = self
+		searchController.searchBar.searchBarStyle = UISearchBarStyle.Minimal
```
Comment (say25) [inline IceFishing/Controllers/LikedTableViewController.swift]: Some of these styling calls are not needed due to StyleController.swift you can see which ones I didn't include on my branch steven/post-history-search

File: IceFishing/Profile/UsersViewController.swift [hunk 2]
```diff
@@ -13,17 +13,42 @@ enum DisplayType: String {
...
 		
         tableView.registerNib(UINib(nibName: "FollowTableViewCell", bundle: nil), forCellReuseIdentifier: "FollowCell")
 		
+		searchController = UISearchController(searchResultsController: nil)
+		searchController.dimsBackgroundDuringPresentation = false
+		searchController.delegate = self
+		searchController.searchResultsUpdater = self
+		
+		//Formating for search Bar
+		searchController.searchBar.sizeToFit()
```
Comment (say25) [inline IceFishing/Profile/UsersViewController.swift]: Some of these styling calls are not needed due to StyleController.swift you can see which ones I didn't include on my branch steven/post-history-search

**Comment by lucasderraugh** [comment]
There are still remaining stylistic issues with the search showing white behind it, but we'll come back to that on touchups.

**Pull request closed without merging**